pyproject.toml

```diff
@@ -16,6 +16,7 @@ edx-opaque-keys = "^2.5.1"
 gitpython = "^3.1.37"
 python-json-logger = "^2.0.2"
 sentry-sdk = "^1.3.1"
+XBlock = "^1.10.0"
```

Comment:
There is a difference between the setup.py generated from pants and the one in the independent repo.

This is the dependencies difference:

**Independent repo:**

```
    install_requires=[
        'django>=2.2,<5.0',
        'XBlock',
        'xblock-utils',
        'edx-opaque-keys'
    ],

```

**This plugin:**
```
    'install_requires': (
        'Django>2.0',
        'XBlock<2.0.0,>=1.10.0',
        'edx-django-utils>4.0.0',
    ),

```

Ideally, we should not have this difference which means we'll probably have to unpin xBlock and add opaque keys package in the pyproject.toml and regenerate the lock file.

Reply:
edx-opaque-keys is already pinned in the pyproject.toml file but pants is not recognizing it as a dependency for rapid-response-xblock. We can define it in the BUILD file as an external dependency: 
```
python_sources(
    name="rapid_response_xblock",
    dependencies=[
        "src/ol_openedx_rapid_response_xblock/migrations:rapid_response_migrations",
        "src/ol_openedx_rapid_response_xblock/settings:rapid_response_settings",
        "//:external_dependencies#edx-opaque-keys"
        ],
)
```

Reply:
That might be because the opaque keys package isn't being used in rapid response. Could you check for its usage?